test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb

```diff
@@ -283,8 +283,8 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
         end
         should "make outcome calculations" do
           assert_state_variable "total_hours", 45
-          assert_state_variable "minimum_hourly_rate", 6.08
-          assert_state_variable "total_hourly_rate", 6.07
+          assert_state_variable "minimum_hourly_rate", 6.5
+          assert_state_variable "total_hourly_rate", "6.12"
```

Comment:
Yeah. There are some similar assertions in this file that have been commented out because of the data being date dependent. The comment suggests that the commented out assertions are now in the calculator test. If we think that's a useful approach then I can imagine doing a similar thing with these values.